frontend-ssr/redux/lesson-comments/lesson-comments.reducer.ts

```diff
@@ -0,0 +1,49 @@
+import { CommentState } from './types';
+import { LESSON_COMMENTS } from './lesson-comments.actions';
+
+const initialState: CommentState = {
+  comments: undefined,
```

Comment:
No pagination, right? Just wanted to make sure.

It's ok, it's not as mandatory as for notifications/submissions since I don't expect a lot of comments in the beginning.